src/Conda.jl

```diff
@@ -35,42 +35,98 @@ using Compat
 import Compat.String
 using JSON
 
-"Prefix for installation of all the packages."
-const PREFIX = abspath(dirname(@__FILE__), "..", "deps", "usr")
+deps_file = joinpath(dirname(@__FILE__), "..", "deps","deps.jl")
+
+if isfile(deps_file)
+    include(deps_file)
+else
+    error("Conda is not properly configured.  Run Pkg.build(\"Conda\") before importing the Conda module.")
+end
+
+typealias Environment Union{AbstractString,Symbol}
+
+RootEnv = default_dir
+
+"Prefix for installation of the environment"
+function prefix(name::Symbol)
+    if (length(string(name)) == 0)
+        return error("Environment name should be non empty.")
```

Comment:
`error` is not a return value.   Just call `error`.   Or `throw(ArgumentError("environment name must be non-empty"))` to have a more specific exception type.